frappe_io/www/docs/user/en/tutorial/setting-up-the-site.md

```diff
@@ -47,13 +47,21 @@ A new folder called `library` will be created in the `sites` folder. Here is the
 1. `private/backups` is where backups are dumped
 1. `site_config.json` is where site level configurations are maintained.
 
-### Setting Default Site
-
-In case you have multiple sites on you bench use `bench use [site_name]` to set the default site.
-
-Example:
-
-	$ bench use library
+### Setting Sites
+
+In case you have multiple sites on your bench , you can map these sites to your localhost by adding your site name to the `/etc/hosts` file in the following manner:
```

Comment:
```suggestion
In case you have multiple sites on your bench, you can map these sites to your localhost by adding your site name to the `/etc/hosts` file in the following manner:
```